libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -4597,8 +4597,8 @@ export const hasCryptoCurrencyId = (id: string): boolean => id in cryptocurrenci
  *
  * @param {*} id
  */
-export function getCryptoCurrencyById(id: string | undefined): CryptoCurrency {
-  const currency = id && findCryptoCurrencyById(id);
```

Comment:
I'm unsure of why we had this `undefined` option for `id`, and that change makes me worried about a nonsensical problem down the road somehow 😆

Reply:
If ever were in the case where `id` is undefined then `currency` would be undefined and then it would throw an error which didn't happened AFAIK

Reply:
This  code change actually has zero functional impact, since `findCryptoCurrencyById` will just return undefined if `id` is actually undefined, and the behaviour of `getCryptoCurrencyById` will be just the same. So should be all good :ok_hand: